lms/templates/admin/instance.html.jinja2

```diff
@@ -42,8 +42,8 @@
 
     <fieldset class="box">
         {{ macros.disabled_text_field("Consumer key", instance.consumer_key) }}
-        {{ macros.disabled_text_field("Deployment ID", instance.deployment_id) }}
-        {{ macros.disabled_text_field("LMS URL", instance.lms_url) }}
+        {{ macros.form_text_field(request, "Deployment ID", "deployment_id", field_value=instance.deployment_id) }}
+        {{ macros.form_text_field(request, "LMS URL", "lms_url", field_value=instance.lms_url) }}
```

Comment:
I thought this was one of the fields we automatically updated on application instances, but it's not. I wonder if we could / should, as this field is probably very wrong in most cases.

Reply:
I removed the option to edit the canvas custom domain, this is overridden in every request from the request parameters.